### ISSUE TITLE
Update to latest version of Yrsa font

### ISSUE THREAD

**Issue by Thomas-N** [google/fonts#1740]
[This issue](https://github.com/rosettatype/yrsa-rasa/issues/12) still shows up on Dutch-language sites using the Yrsa font from Google, although the example linked in that post is no longer relevant (they switched to a font that works properly).

The people developing Yrsa fixed the issue a long time ago, please get google fonts to use the latest version.

Notes by @yanone:
- [x] Contact author
- [x] Build & QA
- [x] [Catalog designer info up to date?](https://github.com/google/fonts/pull/2991)
- [x] [PR to upstream](https://github.com/rosettatype/yrsa-rasa/pull/21)
Currently waiting on reply from @kontur on progress on the repo
- [x] PR to GF
- [x] Font Images

**Comment by thlinard** [comment]
https://github.com/rosettatype/yrsa-rasa

**Comment by yanone** [comment]
PR to upstream repo: https://github.com/rosettatype/yrsa-rasa/pull/21

**Comment by yanone** [comment]
PR to upstream with vertical metrics adjustments: https://github.com/google/fonts/pull/3144
